android/src/main/java/com/aakashns/reactnativedialogs/modules/DialogAndroid.java

```diff
@@ -314,7 +356,36 @@ public void onDismiss(DialogInterface dialog) {
 
             // TODO : Provide pre-selected input types in Javascript
             if (input.hasKey("type")) {
-                mBuilder.inputType(input.getInt("type"));
+                switch (input.getString("type")) {
```

Comment:
I am not sure if you want to make this a breaking change in which case, it would need to be properly documented. We could use a different field for specifying the type using a string (`typeString` maybe) but that will also need to be documented well and the user should be instructed to use either one of them. I will let the collaborators decide on that.